projects/ADuCM3029_demo_cn0428_cn0429/src/ADuCM3029_demo_cn0428_cn0429.cpp

```diff
@@ -0,0 +1,1307 @@
+/**
+******************************************************************************
+*   @file     ADuCM3029_demo_cn0428_cn0429.cpp
+*   @brief    Main project source file
+*   @version  V0.1
+*   @author   ADI
+*
+*******************************************************************************
+*
+*Copyright 2015-2018(c) Analog Devices, Inc.
+*
+*All rights reserved.
+*
+*Redistribution and use in source and binary forms, with or without modification,
+*are permitted provided that the following conditions are met:
+*    - Redistributions of source code must retain the above copyright
+*      notice, this list of conditions and the following disclaimer.
+*    - Redistributions in binary form must reproduce the above copyright
+*      notice, this list of conditions and the following disclaimer in
+*      the documentation and/or other materials provided with the
+*      distribution.
+*    - Neither the name of Analog Devices, Inc. nor the names of its
+*      contributors may be used to endorse or promote products derived
+*      from this software without specific prior written permission.
+*    - The use of this software may or may not infringe the patent rights
+*      of one or more patent holders.  This license does not release you
+*      from the requirement that you obtain separate licenses from these
+*      patent holders to use this software.
+*    - Use of the software either in source or binary form, must be run
+*      on or directly connected to an Analog Devices Inc. component.
+*
+*THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+*INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT, MERCHANTABILITY AND FITNESS FOR A
+*PARTICULAR PURPOSE ARE DISCLAIMED.
+*
+*IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+*EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, INTELLECTUAL PROPERTY
+*RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+*BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+*STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+*THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************
+**/
+
+#include <sys/platform.h>
+#include "adi_initialize.h"
+#include "ADuCM3029_demo_cn0428_cn0429.h"
+#include "adi_m355_gas_sensor.h"
+#include <stdio.h>
+#include <string.h>
+extern "C" {
+#include "RingBuffer.h"
+#include "cli.h"
+}
+
+using namespace adi_sensor_swpack;
+
+/* ================================================== */
+/* 	SYSTEM VARIABLES - handles, memory, error flags	  */
+/* ================================================== */
+ADI_UART_RESULT		  eUartResult;
+ADI_GPIO_RESULT 	  eGpioResult;
+ADI_PWR_RESULT        ePwrResult;
```

Comment:
Looks like the variables are not properly alligned.

Reply:
I will look for issues like this.

Reply:
These still looked perfectly aligned on my machine. Is there an easy way for me to see issues like this?

Reply:
git typically uses 8-tab spaces, so whenever you write code that's a mix of tabs + spaces, you get mixed results ; we also get this even when using just tabs; 
there were many tabs-vs-spaces wars in many SW groups [and some are fun to watch with a bag of pop-corn by your side] as things escalate ;

that's one of the main reasons we're using astyle [which is not a perfect tool btw, but overall it is a good tool for enforcing style, tabs, spaces, etc]

i don't think astyle was the reason for this alignment here [but i could be wrong; i haven't checked]

Reply:
The easiest way to see these issues is to use an editor that allows showing tab/whitespace symbols in your code.  
In the example below I used Notepad++ on your code.

![image](https://user-images.githubusercontent.com/26061529/50335708-d4d93400-0514-11e9-9f92-af04affddf0e.png)

As you can see, at a first glance, the variables are aligned, but there is a mix of tabs and white spaces.